networkx/algorithms/tree/mst.py

```diff
@@ -113,6 +196,8 @@ def prim_mst_edges(G, minimum, weight='weight', keys=True, data=True):
                     yield u, v
 
 ALGORITHMS = {
+    'boruvka': boruvka_mst_edges,
+    'borůvka': boruvka_mst_edges,
```

Comment:
Don't you want this as:   `u"borůvka"`?  Without the `u`, the dictionary only accepts a utf-8 encoded string, not a unicode string.  Perhaps you want both, and then I'd guess we'd have some arbitrary cutoff where we say, we provide ascii (with replacements), utf-8, and unicode, but no other encodings.